homeassistant/components/zha/core/device.py

```diff
@@ -257,16 +282,17 @@ def _check_available(self, *_):
             if difference > _KEEP_ALIVE_INTERVAL:
                 if self._checkins_missed_count < _CHECKIN_GRACE_PERIODS:
                     self._checkins_missed_count += 1
-                    if (
-                        CHANNEL_BASIC in self.cluster_channels
```

Comment:
looks like this check was removed... I know all devices should have them but I swear I hit something that caused me to add this... I could be losing my mind though :)

Reply:
Out of all devices we have between us, only https://github.com/Adminiuga/home-assistant/blob/08bc82de0f33d0f6f1307f408fd72bb320042906/tests/components/zha/zha_devices_list.py#L2468 and next  one have no `Basic` cluster on their 1st endpoint. Both of those devices are of unknown model and manufacturer.

Do you want to add the check in this PR or I could create another one. If we to add this check back, then probably makes sense to treat basic channel the same way as `PowerConfiguration` channel, cause we only need one channel even if the device have more than one

Reply:
I hear ya... we can leave it for now.